### ISSUE TITLE
else is not displayed in context for "if else" in java

### ISSUE THREAD

**Issue by yimingwangdell** [nvim-treesitter/nvim-treesitter-context#376]
### Description

as titled.

### Neovim version

latest

### Actual behavior

have "else" in context

### Minimal config

```Lua
local plugins = {
  ts         = 'https://github.com/nvim-treesitter/nvim-treesitter',
  ts_context = 'https://github.com/nvim-treesitter/nvim-treesitter-context',
  -- ADD ADDITIONAL PLUGINS THAT ARE _NECESSARY_ TO REPRODUCE THE ISSUE
}

for name, url in pairs(plugins) do
  local install_path = '/tmp/nvim/site/'..name
  if vim.fn.isdirectory(install_path) == 0 then
    vim.fn.system { 'git', 'clone', '--depth=1', url, install_path }
  end
  vim.o.runtimepath = install_path..','..vim.o.runtimepath
end

-- ADD INIT.LUA SETTINGS THAT IS _NECESSARY_ FOR REPRODUCING THE ISSUE
```

### Steps to reproduce

for a java file with "if else"

**Comment by yimingwangdell** [comment]
if (xxx ) {
} else {

.. this part is not displayed with the line with "else {" is hidden.
}


if I move cursor here.

**Comment by glbondiii** [comment]
For anyone else who comes here, I'm pretty sure this was marked as "not planned" because it's an issue for the maintainers of the Java Treesitter parser to handle; the Java Treesitter parser does not handle else as its own block (as of 12/24/24), so it could be quite difficult (at best) to make a query that works properly with it, without compromising the functionality of normal if_else. The problem should be trivial to fix once the else block is added separately though (basically copying and pasting from the C query)
For any other parsers with a similar issue, in case you aren't familiar (I honestly didn't know about this until I started looking into this issue), you can run the ":InspectTree" command to get an idea of how Treesitter is parsing through your file, in this case, your else statements. 
TL;DR this project isn't well suited to do anything about this issue; contact those behind Java Treesitter (or for the language that you're having this specific issue with)
